docs/UserGuide.md

```diff
@@ -410,79 +410,120 @@ Deletes the specified delivery from the address book.
-- `s/SUPPLIER_INDEX`: Must be a number greater than 0 and must not be blank.
-- `pro/PRODUCT`: Must be alphanumeric, can include spaces but must not start with a space, and must be between 1 and 50 (inclusive) characters long.
-<box type="tip" seamless>
+- `on/DELIVERY_DATE_TIME`: `DELIVERY_DATE_TIME` must be in dd-mm-yyyy hh:mm format and must not be blank.
+- `stat/STATUS`: `STATUS` must be one of the following: PENDING, DELIVERED, CANCELLED and must not be blank.
+- `s/SUPPLIER_INDEX`: `SUPPLIER_INDEX` must be a number greater than 0 and must not be blank.
+- `pro/PRODUCT`: `PRODUCT` must be alphanumeric, only contain between 1 and 50 (inclusive) characters, and spaces are also allowed.
+    - One space is counted as one character.
+    - `PRODUCT` cannot be made up of only spaces.
+
+</box>
+
+<box type="warning" seamless>
 
 **Warnings**:
-- A spacing between `find` and `-d` is compulsory
-- At least one prefix and parameter must be given
-- No duplicate prefix can be used
-- Find result(s) will contain/satisfy all the given parameters
-- Parameters used are case-insensitive
+- At least one space is needed between `find` and `-d`.
+- At least one parameter must be given.
+- At least one space is needed between `-d` and the first parameter.
+- When using more than one parameter, at least one space is needed between parameters.
+- `STATUS` and `PRODUCT` are not case-sensitive.
+- No duplicate parameter can be used.
+  - `find -d pro/milk pro/bread` is not allowed.
+- Find result(s) will contain/satisfy all the given parameters.
+
 </box>
 
 
-#### Example
-To find deliveries of product "milk" on "28-06-2025 17:00" :
+Example:
+- `find -d on/ 28-06-2025 17:00 pro/ milk`
 
-    find -d on/ 28-06-2025 17:00 pro/ milk
+Expected output:
+- Deliveries of products that contains "milk" on "28-06-2025 17:00", if any, will be displayed.
 
 #### Here's how it would look like in the app:
 ![find command](images/findDeliveryCommand.png)
 
 ### Sort deliveries: `sort -d`
 
-The `sort -d` command is used to sort deliveries in VendorVault.
-This helps you to view the deliveries in a different order, based on the delivery cost, date or status.
+The `sort -d` command is used to sort deliveries in VendorVault based on the delivery cost, date or status.
+This helps you to view the deliveries in a different order (ascending or descending).
 
 Format: `sort -d so/SORT_ORDER sb/SORT_BY`
 
+<box type="details" seamless>
+
 Parameters:
 
-- `SORT_ORDER`: Must be either `a` for ascending or `d` for descending, and must not be blank.
-- `SORT_BY`: Must be either 'c' for cost, `d` for date or `s` for status, and must not be blank.
-<box type="tip" seamless>
+- `so/SORT_ORDER`: `SORT_ORDER` must be either `a` for ascending or `d` for descending, and must not be blank.
+- `sb/SORT_BY`: `SORT_BY` must be either 'c' for cost, `d` for date or `s` for status, and must not be blank.
```

Comment:
Should the cost, date, status be COST, DELIVERY_DATE_TIME and STATUS ? But I think currently cost, date and status makes sense also since the stated params don't appear anywhere in the command usage